cdc/sink/mysql.go

```diff
@@ -1039,7 +1039,8 @@ func (s *mysqlSink) prepareDMLs(rows []*model.RowChangedEvent, replicaID uint64,
 		var args []interface{}
 		quoteTable := quotes.QuoteSchema(row.Table.Schema, row.Table.Table)
 
-		// Translate to UPDATE if old value is enabled, not in safe mode and is update event
+		// If the old value is enabled, is not in safe mode and is an update event, then translate to UPDATE.
```

Comment:
Is it mean `old value enabled && not in safe mode && update event`? 
It is littel bit complicated.

Reply:
Yes